### PR TITLE
fix(NcAvatar): extend variable size of status icons

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

- Fix overlapping of status icons on small-sized avatars

### 🖼️ Screenshots

🏠 Before | 🏡 After
--- | ---
image | image

### 🏁 Checklist

- [ ] ⛑️ Tests are included or are not applicable
- [ ] 📘 Component documentation has been extended, updated or is not applicable
- [ ] 3️⃣ Backport to `next` requested with a Vue 3 upgrade

### CODE REVIEW

File: src/components/NcAvatar/NcAvatar.vue [hunk 1]
```diff
@@ -807,14 +868,14 @@ export default {
...
+		min-height: 14px;
+		min-width: 14px;
 		max-height: 18px;
 		max-width: 18px;
 		height: 40%;
 		width: 40%;
-		line-height: 15px;
-		font-size: var(--default-font-size);
+		line-height: 1;
+		font-size: 85%;
```
Comment (Antreesy) [inline src/components/NcAvatar/NcAvatar.vue:878]:
```suggestion
		font-size: clamp(var(--font-size-small), 85%, var(--default-font-size));
```

**Review by ShGKme — APPROVED** [review]
Thanks

**Review by marcoambrosini — APPROVED** [review]
👌

**Pull request merged**